client/app/bundles/course/assessment/containers/AssessmentForm.intl.js

```diff
@@ -0,0 +1,88 @@
+import { defineMessages } from 'react-intl';
+
+const translations = defineMessages({
+  title: {
+    id: 'course.assessment.form.title',
+    defaultMessage: 'Title',
+  },
+  description: {
+    id: 'course.assessment.form.description',
+    defaultMessage: 'Description',
+  },
+  startAt: {
+    id: 'course.assessment.form.startAt',
+    defaultMessage: 'Start At',
+  },
+  endAt: {
+    id: 'course.assessment.form.endAt',
+    defaultMessage: 'End At',
+  },
+  bonusEndAt: {
+    id: 'course.assessment.form.bonusEndAt',
+    defaultMessage: 'Bonus End At',
+  },
+  baseExp: {
+    id: 'course.assessment.form.baseExp',
+    defaultMessage: 'Base EXP',
+  },
+  timeBonusExp: {
+    id: 'course.assessment.form.timeBonusExp',
+    defaultMessage: 'Time Bonus EXP',
+  },
+  autograded: {
+    id: 'course.assessment.form.autograded',
+    defaultMessage: 'Autograded',
+  },
+  autogradedHint: {
+    id: 'course.assessment.form.autogradedHint',
+    defaultMessage: 'Automatically assign grade and experience points after finalization. \
```

Comment:
I'm not sure the users will understand "finalization", esp new users. Will 
```
Automatically assign grade and experience points after assessment is submitted.
```
be clearer?